INSTALL.doc

```diff
@@ -26,6 +26,12 @@ To produce all the documents, the following tools are needed:
   - convert (ImageMagick)
   - hevea
   - hacha
```

Comment:
Will all the previous dependencies continue to be needed?

Reply:
I don't know. I believe you've worked on other documentation material (tutorial, faq), so you probably know better what is used there.

Reply:
The FAQ is no more (It is now part of the wiki) and the tutorials should follow the same trajectory. Apart from the reference manual, the only other important part of the documentation is generated by coqdoc from the standard library so it depends what the dependencies of coqdoc are.

Reply:
Who can tell me more about this?